sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -77,6 +80,15 @@ private[sql] class DDLParser extends StandardTokenParsers with PackratParsers wi
         CreateTableUsing(tableName, provider, opts)
     }
 
+  /*
+   * describe [extended] table jsonTable
```

Comment:
Nit: `jsonTable` seems a little confusing, remove the comment or add some more detail description.